lighthouse-core/audits/deprecations.js

```diff
@@ -64,6 +64,10 @@ class Deprecations extends Audit {
     let deprecations;
     if (artifacts.InspectorIssues.deprecationIssue.length) {
       deprecations = artifacts.InspectorIssues.deprecationIssue
+        // TODO: translate these strings.
+        // see https://github.com/GoogleChrome/lighthouse/issues/13895
+        // @ts-expect-error: .type hasn't released to npm yet
+        .filter(deprecation => !deprecation.type || deprecation.type === 'Untranslated')
```

Comment:
This is what ToT CDT does right now (but not for much longer, there's a pending CL): just hides these translated strings.